### PR TITLE
Add Javascript examples

### PR DESCRIPTION
**Pull request by NiR-**

### CODE REVIEW

File: examples/angular-realworld-example-app/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+# Finally, you can start the demo
+$ docker-compose -d
+```
+
+You can then browse to http://localhost/.
+
+
+[`jq`]: https://stedolan.github.io/jq/download/
+[`yq`]: https://github.com/kislyuk/yq
```
Comment (nm2107) [inline examples/angular-realworld-example-app/README.md]: :+1:

File: examples/vuejs-realworld-example-app/nginx.conf [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        location /img {
+            alias /app/build/assets;
+
+            expires 1y;
+            access_log off;
+            add_header Cache-Control "public";
+        }
+
+        location / {
+            expires 60s;
```
Comment (nm2107) [inline examples/vuejs-realworld-example-app/nginx.conf]: js and css assets are served via this location for vue js ?

Comment (NiR-) [reply examples/vuejs-realworld-example-app/nginx.conf]: iirc `index.html` is in the root directory and assets are stored in `assets` or in `js/` and `css/`.

Comment (nm2107) [reply examples/vuejs-realworld-example-app/nginx.conf]: should there have also a dedicated nginx location for such assets with an appropriate cache control header (as it is for the angularjs example) ?

**Pull request merged**